app/views/reports/templates/template_1/_footer.html.erb

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+<head>
```

Comment:
<head> should be present in the footer?

Reply:
Yes, I think. The original library accepts only URLs.
`--footer-html <url>             Adds a html footer`
Probably gem running as a static page on the temporary endpoint.